rectools/dataset/interactions.py

```diff
@@ -152,3 +150,42 @@ def get_user_item_matrix(self, include_weights: bool = True) -> sparse.csr_matri
             ),
         )
         return csr
+
+    def to_raw(
+        self,
+        user_id_map: IdMap,
+        item_id_map: IdMap,
+        include_weight: bool = True,
+        include_datetime: bool = True,
+    ) -> pd.DataFrame:
+        """
+        Convert itself to `pd.DataFrame` with replacing internal user and item ids to external ones.
+
+        Parameters
+        ----------
+        user_id_map : IdMap
+            User id map that has to be used for converting internal user ids to external ones.
+        item_id_map : IdMap
+            Item id map that has to be used for converting internal item ids to external ones.
+        include_weight : bool, default ``True``
+            Whether include weight column into resulting dataset or not.
```

Comment:
Needed both here, for next attribute and for dataset class.
```suggestion
            Whether to include weight column into resulting table or not.
```

Reply:
Fixed